device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/transport/mqtt/MqttConnectionTest.java

```diff
@@ -1,547 +0,0 @@
-package tests.unit.com.microsoft.azure.sdk.iot.device.transport.mqtt;
-
-import com.microsoft.azure.sdk.iot.device.ProxySettings;
-import com.microsoft.azure.sdk.iot.device.exceptions.ProtocolException;
-import com.microsoft.azure.sdk.iot.device.transport.HttpProxySocketFactory;
-import com.microsoft.azure.sdk.iot.device.transport.mqtt.MqttConnection;
-import com.microsoft.azure.sdk.iot.device.transport.mqtt.Socks5SocketFactory;
-import mockit.*;
-import org.apache.commons.lang3.tuple.Pair;
-import org.eclipse.paho.client.mqttv3.*;
-import org.eclipse.paho.client.mqttv3.persist.MemoryPersistence;
-import org.junit.Test;
-
-import javax.net.ssl.SSLContext;
-import javax.net.ssl.SSLSocketFactory;
-import java.net.Proxy;
-import java.util.Queue;
-import java.util.concurrent.ConcurrentLinkedQueue;
-
-import static org.junit.Assert.*;
-
-/*
-    Unit test for MqttConnection
-    Coverage : 100% method, 100% line
- */
-public class MqttConnectionTest
```

Comment:
Since the class that this unit tested is now deleted, we can delete this unit test file